test/cmd/help.sh

```diff
@@ -50,6 +50,7 @@ os::cmd::expect_success_and_text 'oadm' 'Install Commands:'
 os::cmd::expect_success_and_text 'oadm ca' 'Manage certificates'
 os::cmd::expect_success_and_text 'openshift start kubernetes' 'Kubernetes server components'
 os::cmd::expect_success_and_text 'oc exec --help' '\[options\] \-\- COMMAND'
+os::cmd::expect_success_and_text 'oc rsh --help' '\[options\] \[COMMAND\]'
```

Comment:
> Yeah, this one is just a short-term improvement but we need to do that.

I just noticed this is the second such problem in two days.  Do you have an issue open?